### PR TITLE
Update TSG.md

### PR DESCRIPTION
**Pull request by Ramyaharinarthini**
Added new error message under Common Mount Problems.

### CODE REVIEW

**Review by vibhansa-msft — APPROVED** [review]

File: TSG.md [hunk 1]
```diff
@@ -105,6 +105,23 @@ For HNS account, always add `type: adls` under `azstorage` section in your confi
...
+
+```
+azstorage:
+  type: adls
+  account-name: <name of the storage account>
+  container: <name of the storage container to be mounted>
+  endpoint: <storage account endpoint (example - https://account-name.blob.core.windows.net)>
+  mode: key
+  account-key: <storage account key>
+```
```
Comment (souravgupta-msft) [inline TSG.md:120]: You can remove this section. The sample config files linked below already mentions this.

File: TSG.md [hunk 2]
```diff
@@ -105,6 +105,23 @@ For HNS account, always add `type: adls` under `azstorage` section in your confi
...
+The [BlobFuse2 base configuration file](https://github.com/Azure/azure-storage-fuse/blob/main/setup/baseConfig.yaml) contains a list of all settings and a brief explanation of each setting.
+
+Use the [sample file cache configuration file](https://github.com/Azure/azure-storage-fuse/blob/main/sampleFileCacheConfig.yaml) or the [sample streaming configuration file](https://github.com/Azure/azure-storage-fuse/blob/main/sampleStreamingConfig.yaml) to get started quickly by using some basic settings for each of those scenarios.
```
Comment (souravgupta-msft) [inline TSG.md:123]: Please update this.
```suggestion
Use the [sample file cache configuration file](https://github.com/Azure/azure-storage-fuse/blob/main/sampleFileCacheConfig.yaml) or the [sample block cache configuration file](https://github.com/Azure/azure-storage-fuse/blob/main/sampleBlockCacheConfig.yaml) to get started quickly by using some basic settings for each of those scenarios.
```

**Comment by vibhansa-msft** [comment]
Kindly update the changeset as per review comments otherwise we will be closing this PR in couple of days.

**Comment by vibhansa-msft** [comment]
Closing this as there are no updates on this. Feel free to reopen once all comments have been addressed.

**Pull request closed without merging**